bot.js

```diff
@@ -32,6 +32,13 @@ Discord.on('message', function (message) {
 	  }
   });
 
+Discord.on('message',  function (message) {
+  const cat = request.get('https://aws.random.cat/meow');

```

Comment:
Doesn't look like request is defined

Reply:
Defined request as require 'snekfetch' and added it to the package.json